src/axolotl/loaders/patch_manager.py

```diff
@@ -471,3 +472,20 @@ def _apply_lora_kernel_patch(self, model):
             from axolotl.monkeypatch.lora_kernels import apply_lora_kernel_patches
 
             apply_lora_kernel_patches(model=model, cfg=self.cfg)
+
+    def _apply_patch_deepspeed_zero3(self):
+        try:
+            from axolotl.monkeypatch.deepspeed_utils import (
+                patch_deepspeed_zero3_missing_attributes,
+            )
+            from transformers.integrations.deepspeed import is_deepspeed_zero3_enabled
+
+            if (
+                is_deepspeed_zero3_enabled()
+                or os.getenv("ACCELERATE_DEEPSPEED_ZERO_STAGE") == "3"
+            ):
+                patch_deepspeed_zero3_missing_attributes()
+        except Exception as e:
```

Comment:
This exception is quite broad. Could you narrow it down to the potential error that would be raised? 

Btw, inside the function, you already have a try catch though, so I'm not sure this is needed.